BackEndApp/v1/views.py

```diff
@@ -299,6 +299,79 @@ def create_project(request):
     return JsonResponse({"success": success, "message": message}, status=status)
 
 
+@api_view(["POST"])
+@is_authenticated
+def clone_project(request):
+    try:
+        username = request.data.get("username")
+        modelLayers = request.data.get("modelLayers")
+        preprocessingParameters = request.data.get("preprocessingParameters")
+        hyperParameters = request.data.get("hyperParameters")
```

Comment:
Please use snake case variable names 
Eg. `model_layers`

Reply:
Yes, I have appropriately changed the case wherever required :D